README.md

```diff
@@ -132,10 +130,24 @@ flutter run adb
 
 ## Contributing
 
-Check our [issues page](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/issues) for tasks to work on.
+Check our [issues page](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/issues?q=is%3Aopen+is%3Aissue+label%3A%22Ready+for+Dev%22) `ready for dev` issues to work on.  Look for issues that are not assigned to somebody.  If you are working on an issue assign yourself to it.  To work on something you must be assigned on the issue. If someone already is, ask if they want help and if they agree add yourself to the assignment. Nobody no in the assignment list should be working on an issue.
```

Comment:
@lukesleeman could you pleae also add here under contributing:

Before you submit a PR, please **ENSURE** that you have run the following commands locally and all the steps pass:
```
flutter clean
flutter format
flutter analyze
flutter test
```
You may want to add a git `pre-commit` hook to do this, see here for more information on [how to use git hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks)

Reply:
Thanks for the suggestion!  I've added it into the readme and will look into using git hooks